### DISCUSSION TITLE
Download Clip ad Download Snapshot dont work on mobile?

### DISCUSSION THREAD

**Discussion by mcdent** [blakeblackshear/frigate#2529] [Ask A Question]
I've noticed that using the HA mobile app, on iOS with Frigate, the download clip/snapshot don't seem to do anything.

This may be by design, or a by-product of that page not being written for mobile, if that's the case, just wondering if there is a way to save a clip/snapshot locally?

(Yes I realise I could do a screen recording or take a screenshot of the local display)

**Comment by blakeblackshear** [comment]
How specifically are you trying to download?

**Reply by mcdent** [reply]
![060D5F59-0E25-4D0E-B3EA-4944220390BE](https://user-images.githubusercontent.com/8769856/147415144-3dee9627-8bec-483b-96e7-f4007142c53d.jpeg)
 Using these buttons/ elements.

**Reply by blakeblackshear** [reply]
It works fine in the Frigate UI directly in Safari. It must be the way the iOS app wraps the web UI.
